nautobot/extras/factory.py

```diff
@@ -0,0 +1,61 @@
+from django.contrib.contenttypes.models import ContentType
+from django.utils.text import slugify
+
+import factory
+import faker
+
+from nautobot.core.factory import OrganizationalModelFactory
+from nautobot.extras.models import Status, Tag
+from nautobot.extras.utils import FeatureQuery, TaggableClassesQuery
+from nautobot.utilities.choices import ColorChoices
+from nautobot.utilities.factory import get_random_instances
+
+
+# Status isn't technically an OrganizationalModel, but it has all of its features **except** dynamic-groups
+class StatusFactory(OrganizationalModelFactory):
```

Comment:
IMO make this comment a docstring instead so that it carries over in IDEs and stuff.